$MirOS: ports/www/dillo/v2/patches/patch-lout_object_cc,v 1.1 2008/11/07 18:05:52 bsiegert Exp $
--- lout/object.cc.orig	Sun Jun  7 05:40:26 2009
+++ lout/object.cc	Sun Aug 23 17:03:44 2009
@@ -153,7 +153,7 @@ int Integer::hashValue()
 void Integer::intoStringBuffer(misc::StringBuffer *sb)
 {
    char buf[64];
-   sprintf(buf, "%d", value);
+   snprintf(buf, sizeof(buf), "%d", value);
    sb->append(buf);
 }
 
